src/commands/publish/upload_form/mod.rs

```diff
@@ -0,0 +1,139 @@
+mod file;
+mod project_assets;
+mod wasm_module;
+
+use log::info;
+
+use reqwest::multipart::{Form, Part};
+use std::fs;
+use std::path::Path;
+
+use crate::commands::build::wranglerjs::Bundle;
```

Comment:
switch this to `use crate::commands::build::wranglerjs` sot hat when `Bundle` appears we know it's `wranglerjs::Bundle`